imap-codec/src/fragmentizer.rs

```diff
@@ -199,19 +205,35 @@ impl Fragmentizer {
         self.max_message_size_exceeded
     }
 
+    /// Returns whether the current message was explicitly poisoned to prevent decoding.
+    pub fn is_message_poisoned(&self) -> bool {
+        self.max_message_size_exceeded
```

Comment:
This should probably be `self.message_poisoned`?

Reply:
Yep. No tests yet.